server/database/models/PictureRepository.js

```diff
@@ -12,6 +12,39 @@ class PictureRepository extends AbstractRepository {
     );
     return rows;
   }
+
+  async create(picture) {
+    const [result] = await this.database.query(
+      `INSERT INTO ${this.table} (image, user_id, art_id) VALUES (?, ?, ?)`,
+      [picture.image, picture.user_id, picture.art_id]
+    );
+
+    return result.insertId;
+  }
+
+  async readByUserId(userId) {
+    const [rows] = await this.database.query(
+      `SELECT * FROM ${this.table} WHERE user_id = ?`,
```

Comment:
Should we not protect the hashed password in here, with this.table of each roww except for the hashed password?

Reply:
Not my task 👍
I'll do a git pull dev once you've put it on dev